### ISSUE TITLE
Site Content goes crazy-town at 2000px

### ISSUE THREAD

**Issue by thisisjoewells** [TheDetroitZoo/ppcc-child#1]
Child theme looks good.

I was just testing this on my uber-big 27-inch iMac.  At exactly 2000px wide the 'site-content' div/section just decides to detach from the rest of the website.  Up to 1999 is fine.

I'm sure few will ever try to be 2000px wide...but...thoughts?

Perhaps this will be hard to even duplicate on a laptop.

![polk-at-200px-wide-wonky](https://cloud.githubusercontent.com/assets/9747248/12919159/c9047f0a-cf10-11e5-9af4-9ae01d71a4ec.jpg)

**Comment by joelworsham** [comment]
Holy cow... sorry about this. I will get this resolved ASAP

**Comment by joelworsham** [comment]
I don't think this is happening anymore, probably because of the centering on desktop change (#7). Can you confirm or deny this?

**Comment by thisisjoewells** [comment]
Yes.  The crazy-town is fixed along with centering on desktop.  Feel free to push to live (when we go live of course).
